lib/srv/monitor.go

```
@@ -194,6 +201,10 @@ func (w *Monitor) Start() {
 						now.Sub(clientLastActive), w.ClientIdleTimeout)
 				}
 				w.Entry.Debugf("Disconnecting client: %v", event.Reason)
+
+				if w.MessageWriter != nil && w.IdleTimeoutMessage != "" {
+					w.MessageWriter.WriteString(w.IdleTimeoutMessage)
```

Do not ignore the error.

---

Logged